Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10.19.0-stretch
```

Comment:
You might want to use `[buildkite/puppeteer`](https://hub.docker.com/r/buildkite/puppeteer/tags/)` as I seem to recall it's a real pain getting the `node` image to run Puppeteer. (which Pa11y needs)

Reply:
True, puppeter is needed, but did a try on buildkite/puppeteer - but it did not work out with the sidekick code is it is right now - but I will look into it -I will be back on the subject.

Reply:
I updated Dockerfile with another setup that I am using with puppeteer, using alpine.

Reply:
Or rather - using puppeteer-core